.travis-install.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/sh
```

Comment:
Should we put all this inside the before_install step inside the original .travis.yml file ?